samples/quickstart.js

```diff
@@ -0,0 +1,143 @@
+/**
+ * Copyright 2020, Google LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
```

Comment:
Can we add a sample header here with title, description, and usage info?  For example:

```

// sample-metadata:
//   title: Export Assets
//   description: Export asserts to specified dump file path.
//   usage: node exportAssets.js <gs://my-bucket/my-assets.txt>
```